package.json

```diff
@@ -20,33 +21,50 @@
   },
   "homepage": "https://github.com/kids-first/kf-arranger#readme",
   "dependencies": {
-    "@kfarranger/server": "^1.2.2",
+    "@kfarranger/server": "^1.2.3",
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
+    "@babel/plugin-proposal-optional-chaining": "^7.2.0",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/register": "^7.5.5",
     "apollo-server-core": "1.3.2",
-    "babel-polyfill": "^6.26.0",
     "body-parser": "^1.18.2",
     "chalk-animation": "^1.5.0",
+    "core-js": "^3.1.4",
```

Comment:
Haven't used this one yet- hopefully it remains supported / open for a long time.

Reply:
Replaces `babel-polyfill`.
see https://babeljs.io/docs/en/7.4.0/babel-polyfill